### PR TITLE
fix: skills page styles #170, #175

### PR DESCRIPTION
**Pull request by twistedomelette**

### CODE REVIEW

File: frontend/src/components/skills/skill-overview.tsx [hunk 1]
```diff
@@ -184,7 +184,6 @@ const SkillOverview = (): React.ReactElement => {
             >
               <option>All Skills</option>
               <option value={`${IMPORTANT}`}>Important Skills</option>
-              <option value={`${UNIMPORTANT}`}>Unimportant Skills</option>
```
Comment (nikita-remeslov) [inline frontend/src/components/skills/skill-overview.tsx:187]: Why did you remove this?

File: frontend/src/components/skills/skill-overview.tsx [hunk 2]
```diff
@@ -184,7 +184,6 @@ const SkillOverview = (): React.ReactElement => {
             >
               <option>All Skills</option>
```
Comment (nikita-remeslov) [inline frontend/src/components/skills/skill-overview.tsx:185]: Please left 

> All
> Important
> Unimportant

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/components/skills/skill-overview.tsx (hunk 1 above)
Comment (twistedomelette) [reply frontend/src/components/skills/skill-overview.tsx:187]: I had a ticket with this task

File: frontend/src/components/skills/skill-overview.tsx (hunk 2 above)
Comment (twistedomelette) [reply frontend/src/components/skills/skill-overview.tsx:185]: Ok
